docs/hour-of-code/educators-2021.html

```diff
@@ -550,6 +550,22 @@ <h3 class="display-font title sunflower">
                     </li>
                 </ul>
             </section>
+    
+            <section class="hoc-section">
+                <h3 class="display-font title sunflower">
+                    MakeCode Insiders Club
+                </h3>
+                <p>
+                    Help us find engaged teens interested in influencing 
```

Comment:
"Help us find teens" doesn't sound so good...  How about changing to, "Do you know a student who may be interested in helping to influence product development?  The Microsoft MakeCode team is recruiting members for its upcoming MakeCode Insiders Club Winter Cohort."

Reply:
Just made those changes! Thanks!